ReactWindows/ReactNative.Shared/Modules/Wrappers/UIManagerModuleWrapper.cs

```diff
@@ -74,100 +74,100 @@ public void OnBatchComplete()
 
         private void ConfigureNextAnimationLayout(IReactInstance instance, JArray arguments)
         {
-            _module.configureNextLayoutAnimation((JObject)arguments[0], new Callback(arguments[1].Value<int>(), instance), new Callback(arguments[2].Value<int>(), instance));
+            _module.configureNextLayoutAnimation((JObject)arguments[0], new Callback(ToInt32(arguments[1]), instance), new Callback(ToInt32(arguments[2]), instance));
         }
 
         private void CreateView(IReactInstance instance, JArray arguments)
         {
-            _module.createView(arguments[0].Value<int>(), arguments[1].Value<string>(), arguments[2].Value<int>(), (JObject)arguments[3]);
+            _module.createView(ToInt32(arguments[0]), ToString(arguments[1]), ToInt32(arguments[2]), (JObject)arguments[3]);
         }
 
         private void DispatchViewManagerCommand(IReactInstance instance, JArray arguments)
         {
-            _module.dispatchViewManagerCommand(arguments[0].Value<int>(), arguments[1].Value<int>(), (JArray)arguments[2]);
+            _module.dispatchViewManagerCommand(ToInt32(arguments[0]), ToInt32(arguments[1]), (JArray)arguments[2]);
         }
 
         private void FindSubviewIn(IReactInstance instance, JArray arguments)
         {
-            _module.findSubviewIn(arguments[0].Value<int>(), (JArray)arguments[1], new Callback(arguments[2].Value<int>(), instance));
+            _module.findSubviewIn(ToInt32(arguments[0]), (JArray)arguments[1], new Callback(ToInt32(arguments[2]), instance));
         }
 
         private void ManageChildren(IReactInstance instance, JArray arguments)
         {
             _module.manageChildren(
-                arguments[0].Value<int>(), 
-                ToValueArray<int>((JArray) arguments[1]),
-                ToValueArray<int>((JArray) arguments[2]), 
-                ToValueArray<int>((JArray) arguments[3]),
-                ToValueArray<int>((JArray) arguments[4]), 
-                ToValueArray<int>((JArray) arguments[5]));
+                ToInt32(arguments[0]), 
+                ToValueArray(arguments[1], ToInt32),
+                ToValueArray(arguments[2], ToInt32), 
+                ToValueArray(arguments[3], ToInt32),
+                ToValueArray(arguments[4], ToInt32), 
+                ToValueArray(arguments[5], ToInt32));
```

Comment:
These args can be null. Perhaps ToValueArray could just return null in such cases.